cli/tools/bench/mod.rs

```diff
@@ -421,7 +421,7 @@ pub async fn run_benchmarks_with_watch(
       let bench_flags = bench_flags.clone();
       Ok(async move {
         let factory = CliFactoryBuilder::new()
-          .with_watcher(sender.clone())
+          .with_watcher(sender.clone(), None)
```

Comment:
I think I will refactor this so that `with_watcher()` accepts an options struct instead of two free args